client.lua

```diff
@@ -37,6 +38,16 @@ RegisterNetEvent('QBCore:Client:OnPlayerUnload', function()
     PlayerData = {}
 end)
 
+--- Draws Text onto the screen during test drive
```

Comment:
```suggestion
--- Draws Text onto the screen
```
While the context of where this is called might currently be during a test drive, the function itself is independent and could be called from a different context.